ci-operator/jobs/maistra/istio/maistra-istio-maistra-2.4-presubmits.yaml

```diff
@@ -213,6 +213,69 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )maistra-istio-lint-2-4,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^maistra-2\.4$
+    - ^maistra-2\.4-
+    cluster: build03
+    context: ci/prow/maistra-istio-push-image-2-4
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-maistra-istio-maistra-2.4-maistra-istio-push-image-2-4
+    rerun_command: /test maistra-istio-push-image-2-4
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --hive-kubeconfig=/secrets/hive-hive-credentials/kubeconfig
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --report-credentials-file=/etc/report/credentials
+        - --secret-dir=/secrets/ci-pull-credentials
+        - --target=maistra-istio-push-image-2-4
+        command:
+        - ci-operator
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /secrets/ci-pull-credentials
+          name: ci-pull-credentials
+          readOnly: true
+        - mountPath: /secrets/gcs
+          name: gcs-credentials
+          readOnly: true
+        - mountPath: /secrets/hive-hive-credentials
+          name: hive-hive-credentials
+          readOnly: true
+        - mountPath: /etc/pull-secret
+          name: pull-secret
+          readOnly: true
+        - mountPath: /etc/report
+          name: result-aggregator
+          readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+      - name: ci-pull-credentials
+        secret:
+          secretName: ci-pull-credentials
+      - name: hive-hive-credentials
+        secret:
+          secretName: hive-hive-credentials
+      - name: pull-secret
+        secret:
+          secretName: registry-pull-credentials
+      - name: result-aggregator
+        secret:
+          secretName: result-aggregator
+    trigger: (?m)^/test( | .* )maistra-istio-push-image-2-4,?($|\s.*)
```

Comment:
I wonder if we can't simplify the triggers to  be more action-oriented. It's not really `/test action` but `/action`. So maybe `/push-image` would be easier to guess :) WDYT @yxun

Reply:
@bartoszmajsak  Thanks for your review. The files under ci-operator/jobs/ are auto generated from openshift-ci based on the files under ci-operator/config/ 
We don't have control of jobs/ files. They will be overwritten automatically in one of the presubmit job in each openshift CI PR. 

I can check more if we can customize their trigger config in a prow plugin setting later.